### DISCUSSION TITLE
Disable functions as sitdown

### DISCUSSION THREAD

**Discussion by soyochanz** [QI-Software/deadmatterpublic#138] [Ideas]
Disable them until they are working properly to avoid ppl getting glitched, also with the ability to jump through windows because sometimes i got stuck

**Comment by johnnyguitarFP** [comment]
Possibly for furniture, but absolutely not for climbing through windows. 

It's far too important to the gameplay of the game to disable that, unlike stuff like leaning or melee bash where they were flat out broken and gave you an advantage or broken giving you zero advantage as an extra addition, climbing through windows is a pretty critical component of navigating the in-game world.

**Comment by johnnyguitarFP** [comment]
disabled for furniture, windows use the new vaulting.
